test/integration/wallet.test.js

```diff
@@ -260,7 +260,7 @@ describe('@integration: Wallet', () => {
       step(`Then I should see text ${successMessage} in "result box message" element`, () => helper.haveTextOf('.result-box-message', successMessage));
     });
 
-    describe('Scenario: should allow to send LSK from unlocked account with second passphrase', () => {
+    describe.skip('Scenario: should allow to send LSK from unlocked account with second passphrase', () => {
```

Comment:
Please add a comment saying "This should be unskipped in #<ISSUE_NUMBER>"

Reply:
Comment fixed with issue number added